_posts/2017-10-31-ciw-paper.md

```diff
@@ -0,0 +1,86 @@
+---
+layout     : post
+title      : "Reprodicibility in Discrete Event Simulations"
+language   : english
+---
+
+There is a
+[reproducibility crisis](http://www.bbc.co.uk/news/science-environment-39054778)
+in science.
+
+In computational science this is completely avoidable.
+If we can tell a computer what to do, we can tell it to do it again.
+So we can tell other people how to tell their computer to do it again.
+
+In the domain of discrete event simulation, the software and methodologies that
+are traditionally used can make it difficult to this.
+In a recent preprint I have submitted with
+[Dr. Vincent Knight](http://vknight.org/),
+[Prof. Paul Harper](http://www.profpaulharper.com/), and
+[Asyl Hawa](http://asylhawa.com/), we compare simulation approaches in terms of
+reproducibility and best practice, and propose a Python library for discrete
```

Comment:
`s/compare/introduce`? (You do more than propose: propose would be "it would be cool if someone did this" but you've actually done this.